zerver/tests/fixtures/slack_message_conversion.json

```diff
@@ -84,6 +99,11 @@
       "name": "italic_and_strike_conversion",
       "input": "_~italic~_ and ~_strike_~",
       "conversion_output": "*~~italic~~* and ~~*strike*~~"
+    },
+    {
+      "name": "unsupported_bold_test",
```

Comment:
This will also fail using the old reformatter, Slack incoming webhook's regex was able to format this string properly, but fail at other tests here, the trade-off is detailed here #31336.